modal/_container_entrypoint.py

```diff
@@ -178,11 +179,15 @@ async def _generate_inputs(
     ) -> AsyncIterator[tuple[str, api_pb2.FunctionInput]]:
         request = api_pb2.FunctionGetInputsRequest(function_id=self.function_id)
         eof_received = False
+        iteration = 0
         while not eof_received:
             request.average_call_time = self.get_average_call_time()
             request.max_values = self.get_max_inputs_to_fetch()  # Deprecated; remove.
 
             with trace("get_inputs"):
+                span = tracer.current_span()
```

Comment:
Not sure how ddtrace works, but I assume this is safe to call even when the tracing is disabled?

Reply:
derp nej, inte i python 👎

Reply:
ptal again